kasa/device_factory.py

```diff
@@ -201,12 +205,11 @@ def get_protocol(
     ] = {
         "IOT.XOR": (IotProtocol, XorTransport),
         "IOT.KLAP": (IotProtocol, KlapTransport),
-        "IOT.XOR.HTTPS.2": (IotProtocol, LinkieTransportV2),
         "SMART.AES": (SmartProtocol, AesTransport),
-        "SMART.AES.2": (SmartProtocol, AesTransport),
-        "SMART.KLAP.2": (SmartProtocol, KlapTransportV2),
-        "SMART.AES.HTTPS.2": (SmartCamProtocol, SslAesTransport),
-        "SMART.AES.HTTPS": (SmartProtocol, SslTransport),
+        "SMART.KLAP": (SmartProtocol, KlapTransportV2),
+        # Still require a lookup for SslAesTransport as H200 has a type of
+        # SMART.TAPOHUB.
```

Comment:
I don't fully understand this comment, is this key not generally valid or..?

Reply:
Comment updated